src/OpenTelemetry/AssemblyInfo.cs

```diff
@@ -17,6 +17,7 @@
 using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("OpenTelemetry.Tests" + AssemblyInfo.PublicKey)]
+[assembly: InternalsVisibleTo("OpenTelemetry.Exporter.InMemory" + AssemblyInfo.PublicKey)]
```

Comment:
Why is this needed?

Reply:
When making a copy of `Metric`, `InstrumentIdentity` is internal.

https://github.com/open-telemetry/opentelemetry-dotnet/blob/d1ec45ce15cfe3062f828706bdd883f6ebcc5ded/src/OpenTelemetry/Metrics/Metric.cs#L127

`ExportableMetricCopy` needs access to this. It's referenced 80+ times in tests.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/afb1d863c740aed3d326bf16df45ed093bb2736c/src/OpenTelemetry.Exporter.InMemory/ExportableMetricCopy.cs#L30-L42